### ISSUE TITLE
Timestamp format Error should be a Warning

### ISSUE THREAD

**Issue by james-woods** [nielsbasjes/logparser#49]
This error message should be a warning:
```
The timestamp format "%F %H:%M:%S.%usec_frac" does NOT contain a timezone so we assume "Z".
```

Otherwise, it can have the effect that a pipeline gets canceled since it looks like something really bad happened.

**Comment by nielsbasjes** [comment]
Fixed. Pushing new snapshot right now.

**Comment by james-woods** [comment]
Great! Do you think a 4.1 release in the next couple of weeks is realistic?

**Comment by nielsbasjes** [comment]
Yes, it is going to be 5.0 because of some API changes I made to support a much more fluent API.
I just pushed 5.0-SNAPSHOT so you can try it out.

**Comment by nielsbasjes** [comment]
I just released 5.0.
Happy parsing.
